hollow/src/main/java/com/netflix/hollow/core/write/objectmapper/HollowListTypeMapper.java

```diff
@@ -69,7 +72,7 @@ public int write(Object obj) {
         if(ignoreListOrdering) {
             IntList ordinalList = getIntList();
             for(Object o : l) {
-                Objects.requireNonNull(o, "Null element. Lists cannot contain null elements");
+                Objects.requireNonNull(o, String.format(NULL_ELEMENT_MESSAGE, schema));
```

Comment:
Looks good, although I wonder if it's worth moving the String.format outside of the loop - no need to create that string once per list element.

Reply:
Doh! good point. Will fix.